File: Dart/Encapsulamento/Escola/aluno.dart
class Aluno {
  // Atributos da classe Aluno
  String _nome = "";
  String _cpf = "";
  String _matricula = "";

  // Construtor
  Aluno(this._nome) {}

  // Normalmente, só se usa chaves quando o setter/getter tem mais de uma linha de código.
  // Para uma linha só, pode-se usar arrow function.

  void set setCpf(String cpfEspecifico) => _cpf = cpfEspecifico;

  // Getters precisam ser declarados sem parâmetros.
  // Assim, não precisam de parêntesese em Dart.
  String get getCpf => _cpf;

  void set getMattricula(String matriculaEspecifica) =>
      _matricula = matriculaEspecifica;

  String get getMatricula => _matricula;

  // Método que retorna as informações
  String retornarInformacoesAluno() { 
    return "\nAluno \nNome: $_nome \nCPF: $_cpf \nMatrícula: $_matricula\n";
  }
}

File: linguagem_java/encapsulamento/escola/Aluno.java
package encapsulamento.escola;

/**
 * Aluno
 */
public class Aluno {
    // Atributos da classe Aluno
    // Em Java, um dos modificadores de acesso que podem ser usados
    // para encapsulamento é 'private'
    private String nome;
    private String cpf;
    private String matricula;


    // Construtor
    Aluno(String nomeEspecifico){ 
        setNome(nomeEspecifico);
    }

    // Getter e setter de CPF
    void setCpf(String cpfEspecifico){
        this.cpf = cpfEspecifico;
    }

    String getCpf(){ 
        return this.cpf;
    }

    // Getter e setter de matrícula
    void setMatricula(String matriculaEspecifica){ 
        this.matricula = matriculaEspecifica;
    }

    String getMatricula(){ 
        return this.matricula;
    }

    // Getter e setter de nome
    void setNome(String nomeEspecifico){ 
        this.nome = nomeEspecifico;
    }

    String getNome(){ 
        return this.nome;
    }
}